frontend/pages/register.vue

```diff
@@ -90,49 +89,95 @@ async function registerUser(userData: Object) {
 
 <template>
   <Toast />
-  <div class="w-full flex justify-center items-center font-nunito dark:bg-background-dark">
+  <div
+    class="w-full flex justify-center items-center font-nunito dark:bg-background-dark"
+  >
     <div class="xl:w-1/3 md:w-1/4 sm:w-0 h-[90vh] dark:background-dark">
-      <SvgAircraft class="xl:w-[230%] md:w-[350%] w-0 object-none -ml-[20vw] overflow-hidden mt-16 z-0" />
+      <SvgAircraft
+        class="xl:w-[230%] md:w-[350%] w-0 object-none -ml-[20vw] overflow-hidden mt-16 z-0"
+      />
     </div>
-    <div class="flex xl:w-1/3 md:w-2/4 sm:w-full items-center h-[90vh] justify-center w-full">
-      <div class="text-center mt-6 sm:w-3/4 w-full h-3/4 z-20">
-        <fieldset id="outerBlock"
-          class="h-auto px-3 py-2 bg-surface dark:bg-surface-dark rounded-3xl border-border border-2 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 flex flex-col items-center">
-          <legend for="outerBlock"
-            class="text-3xl text-center lg:text-left lg:ml-7 px-2 font-bold text-text dark:text-white">
+    <div
+      class="flex xl:w-1/3 md:w-2/4 sm:w-full items-center h-[90vh] justify-center w-full"
+    >
+      <div class="text-center sm:w-3/4 w-full -mt-20 h-3/4 z-20">
+        <fieldset
+          id="outerBlock"
+          class="h-auto px-3 py-2 bg-surface dark:bg-surface-dark rounded-3xl border-border border-2 shadow-sm focus:outline-none focus:ring-indigo-500 focus:border-indigo-500 flex flex-col items-center"
+        >
+          <legend
+            for="outerBlock"
+            class="text-3xl text-center lg:text-left lg:ml-7 px-2 font-bold text-text dark:text-white"
+          >
             <T keyName="form.header.register" />
           </legend>
           <form @submit="onSubmit" class="w-4/5">
-            <FormInput id="email" name="email" translationKey="form.input.email" />
+            <FormInput
+              id="email"
+              name="email"
+              autocomplete="email"
+              translationKey="form.input.email"
+            />
 
-            <FormInput id="name" name="name" translationKey="form.input.username" />
+            <FormInput
+              id="firstName"
+              name="firstName"
+              autocomplete="given-name"
+              translationKey="form.input.first.name"
+            />
 
-            <FormPassword id="password" name="password" :feedback="true" :feedbackStyle="true"
-              translationKey="form.input.password" />
+            <FormInput
+              id="lastName"
```

Comment:
Ich weiß nicht aber vielleicht sollte man neben Nachname auch so ein in klammern ein "(optional)" hin machen, damit der User weiß, dass er nicht seinen Nachnamen eintippen muss.